ProfileManager.js

```diff
@@ -229,8 +217,8 @@ export default class ProfileManager {
    *   the EDV; either this or an EDV ID must be given.
    * @param {object} [options.revocationCapability] - The capability to use to
    *   revoke delegated EDV zcaps.
-   * @param {string} options.capabilityInvoker - The type of invoker to sign
-   * capability invocations.
+   * @param {boolean} options.useEphemeralSigner - Flag to enable invoking
+   *  capabilities with the ephemeral invocation signer, default `true`.
```

Comment:
We can give a bit more detail here and everywhere else we talk about this flag and it would help... I think (also fixes indent off-by-1 space):

```suggestion
   *   capabilities with the ephemeral invocation signer associated with the
   *   currently authenticated session, default `true`.
```

Reply:
Updated all docs for `useEphemeralSigner` with proposed text.